perf.groovy

```diff
@@ -0,0 +1,129 @@
+                  'Ubuntu16.04':'Linux',
+                  'Debian8.4':'Linux',
+                  'Fedora23':'Linux',
+                  'OSX':'OSX',
+                  'Windows_NT':'Windows_NT',
+                  'CentOS7.1': 'Linux',
+                  'OpenSUSE13.2': 'Linux',
+                  'RHEL7.2': 'Linux',
+                  'LinuxARMEmulator': 'Linux']
+
+// Map of os -> nuget runtime
+def targetNugetRuntimeMap = ['OSX' : 'osx.10.10-x64',
+                             'Ubuntu14.04' : 'ubuntu.14.04-x64',
+                             'Ubuntu16.04' : 'ubuntu.16.04-x64',
+                             'Fedora23' : 'fedora.23-x64',
+                             'Debian8.4' : 'debian.8-x64',
+                             'CentOS7.1' : 'centos.7-x64',
+                             'OpenSUSE13.2' : 'opensuse.13.2-x64',
+                             'RHEL7.2': 'rhel.7-x64']
+
+def osShortName = ['Windows 10': 'win10',
+                   'Windows 7' : 'win7',
+                   'Windows_NT' : 'windows_nt',
+                   'Ubuntu14.04' : 'ubuntu14.04',
+                   'OSX' : 'osx',
+                   'Windows Nano 2016' : 'winnano16',
+                   'Ubuntu16.04' : 'ubuntu16.04',
+                   'CentOS7.1' : 'centos7.1',
+                   'Debian8.4' : 'debian8.4',
+                   'OpenSUSE13.2' : 'opensuse13.2',
+                   'Fedora23' : 'fedora23',
+                   'RHEL7.2' : 'rhel7.2']
+
+
+// **************************
+// Define perf testing.  These tests should be run daily, and will run on Jenkins not in Helix
+// **************************
+[true, false].each { isPR ->
+    ['Release'].each { configurationGroup ->
+        ['Windows_NT'].each { os ->
+            def osGroup = osGroupMap[os]
+            def newJobName = "perf_${os.toLowerCase()}_${configurationGroup.toLowerCase()}"
+
+            def newJob = job(Utilities.getFullJobName(project, newJobName, isPR)) {
+            
+                label('windows_clr_perf')
+                wrappers {
+                    credentialsBinding {
+                        string('BV_UPLOAD_SAS_TOKEN', 'CoreFX Perf BenchView Sas')
+                    }
+                }
+
+                if (isPR)
+                {
+                    parameters
+                    {
+                        stringParam('BenchviewCommitName', '\${ghprbPullTitle}', 'The name that you will be used to build the full title of a run in Benchview.  The final name will be of the form <branch> private BenchviewCommitName')
+                    }
+                }
+                def configuration = 'Release'
+                def runType = isPR ? 'private' : 'rolling'
+                def benchViewName = isPR ? 'corefx private %BenchviewCommitName%' : 'corefx rolling %GIT_BRANCH_WITHOUT_ORIGIN% %GIT_COMMIT%'
+
+                steps {
+                    //We need to specify the max cpu count to be one as we do not want to be executing performance tests in parallel
```

Comment:
Not sure where you specify CPU count, but the harness won't allow you to run in parallel.  It serializes all execution.